### PR TITLE
Add Blur and remove effects to VideoEffectsPane

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Add Blur and remove effects to VideoEffectsPane
Screenshot 2023-03-29 at 1 16 59 PM

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3204831

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by edwardlee-msft — APPROVED** [review]

File: packages/react-composites/src/composites/common/VideoEffectsPane.tsx [hunk 1]
```diff
@@ -12,13 +20,63 @@ import { Panel } from '@fluentui/react';
 export const VideoEffectsPane = (props: {
   showVideoEffectsOptions: boolean;
   setshowVideoEffectsOptions: (showVideoEffectsOptions: boolean) => void;
+  adapter: CallAdapter | CommonCallAdapter;
 }): JSX.Element => {
-  return VideoEffectsPaneTrampoline(props.showVideoEffectsOptions, props.setshowVideoEffectsOptions);
+  const { showVideoEffectsOptions, setshowVideoEffectsOptions } = props;
+  /* @conditional-compile-remove(video-background-effects) */
```
Comment (mgamis-msft) [inline packages/react-composites/src/composites/common/VideoEffectsPane.tsx:26]: Is this an internal component that will only be used for the video-background-effects feature? If yes, I think we don't need the conditional compilation statements.

File: packages/react-composites/src/composites/CallComposite/Strings.tsx [hunk 2]
```diff
@@ -314,6 +314,26 @@ export interface CallCompositeStrings {
...
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Tooltip text for the blur video background effect item
+   */
+  blurBackgroundTooltip?: string;
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Label for the remove video background effect item
+   */
+  removeBackgroundEffect?: string;
```
Comment (mgamis-msft) [inline packages/react-composites/src/composites/CallComposite/Strings.tsx:331]: is removeBackgroundEffectButtonLabel more descriptive?

Comment (prabhjot-msft) [reply packages/react-composites/src/composites/CallComposite/Strings.tsx]: done

File: packages/react-composites/src/composites/common/VideoEffectsPane.tsx (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-composites/src/composites/common/VideoEffectsPane.tsx:26]: But you are already not showing the VideoEffectsPane in stable by having a CC statement in CallWithChatComposite.tsx
image

Comment (mgamis-msft) [reply packages/react-composites/src/composites/common/VideoEffectsPane.tsx:26]: I'll still approve to unblock :) but I still think the CC lines are unnecessary

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-composites/src/composites/common/VideoEffectsPane.tsx (hunk 1 above)
Comment (prabhjot-msft) [reply packages/react-composites/src/composites/common/VideoEffectsPane.tsx:26]: Thanks Miguel, for pointing this out. I will merge this PR to get unblocked and have created a separate item to remove the CC from video effects pane https://skype.visualstudio.com/SPOOL/_workitems/edit/3207057

**Pull request merged**